pytest/tests/adversarial/fork_sync.py

```diff
@@ -64,26 +64,20 @@
 logger.info("all nodes restarted")
 
 while cur_height < TIMEOUT:
```

Comment:
comment on what this while loop is testing

Reply:
I mean… How should I know. I didn’t write the test. ;)